roles/falcon_configure/README.md

```diff
@@ -40,6 +39,13 @@ future.
 - `falcon_windows_become_method` - The way to become a privileged user on Windows (string, default: ***runas***)
 - `falcon_windows_become_user` - The privileged user to install the sensor on Windows (string, default: ***SYSTEM***)
 
+### macOS Specific Variables
+
+- `falcon_option_set` - Set True|yes to set options, False|no to delete. *See note below (bool, default: ***true***)
```

Comment:
Confirming you wanted an asterisk here (`*See`) and that that isn't the mistaken beginning of an italics.

Reply:
I fixed it.. it did not need to be there.. and yes, it is for an asterik